README.adoc

```diff
@@ -269,6 +274,78 @@ properties are automatically stored in the inventory. Go back to http://localhos
 you see a new entry for `[system-ip-address]`. 
 
 
+== Externalizing server configuration
+
+// File 0
+inventory/server.xml
+[source, xml, linenums, indent=0, role="code_column"]
+----
+include::finish/inventory/src/main/liberty/config/server.xml[]
+----
+
+As mentioned in the beginning of this guide, one of the advantages of using
+containers is that they are portable and can be moved and deployed efficiently
+across all of your DevOps environments. Configuration often changes across
+different environments, and by externalizing your server's configuration, you
+can simplify the development process.
+
+Imagine a scenario where you are developing an Open Liberty application on
+port `9081` but when it comes time to deploy it to production, it must be available
+on port `9091`. To manage this, you could keep two different versions of the
+`server.xml` file; one for production and one for development. However, trying to
+maintain two different versions of a file may lead to mistakes. A better
+solution would be to externalize the configuration of the port number and use the
+value of an environment variable that is stored in each environment. 
+
+In this example, you will use an environment variable to externally configure the
+HTTP port number of the `inventory` service. 
+
+In the [hotspot file=0]`inventory/server.xml` file, the [hotspot=httpPort
+file=0]`default.http.port` variable is declared and is used in the
+[hotspot=httpEndpoint file=0]`httpEndpoint` element to define the service's
+endpoint. The default value of the [hotspot=httpPort file=0]`default.http.port`
+variable is `9081`, however this value is only used if no other value is
+specified. To find a value for this variable, Open Liberty will look for the
```

Comment:
Change,
"To find a value for this variable, Open Liberty will look for the
following environment variables, in order:"
to,
"To find a value for this variable, Open Liberty looks for the
following environment variables, in order:"